### DISCUSSION TITLE
Support InverseForm Loss 支持 InverseForm Loss

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#522] [MMSegmentation]
# Support InverseForm Loss 
 支持 InverseForm Loss 

Issue of the request.
 [[https://github.com/open-mmlab/mmsegmentation/issues/3159](https://github.com/open-mmlab/mmsegmentation/issues/3159)](https://github.com/open-mmlab/mmsegmentation/issues/3159](https://github.com/open-mmlab/mmsegmentation/issues/3159)) 
 
 Reference repo: [[https://github.com/Qualcomm-AI-research/InverseForm](https://github.com/Qualcomm-AI-research/InverseForm)](https://github.com/Qualcomm-AI-research/InverseForm](https://github.com/Qualcomm-AI-research/InverseForm)) 
 
 Reference paper: [[https://arxiv.org/abs/2104.02745](https://arxiv.org/abs/2104.02745)](https://arxiv.org/abs/2104.02745](https://arxiv.org/abs/2104.02745)) 

提出需求的 issue:
 [[https://github.com/open-mmlab/mmsegmentation/issues/3159](https://github.com/open-mmlab/mmsegmentation/issues/3159)](https://github.com/open-mmlab/mmsegmentation/issues/3159](https://github.com/open-mmlab/mmsegmentation/issues/3159)) 
 
 参考 repo :[[https://github.com/Qualcomm-AI-research/InverseForm](https://github.com/Qualcomm-AI-research/InverseForm)](https://github.com/Qualcomm-AI-research/InverseForm](https://github.com/Qualcomm-AI-research/InverseForm)) 
 
 参考论文：[[https://arxiv.org/abs/2104.02745](https://arxiv.org/abs/2104.02745)](https://arxiv.org/abs/2104.02745](https://arxiv.org/abs/2104.02745)) 

## Task Information

**Repository**：MMSegmentation

**Difficulty**：Hard

**Points**：50

**Estimated Time Consumption**：28 days

## Task Status

**Claimant's Github ID**：CastleDream

**Claim Time**：2023年08月03日

**PR Link**：https://github.com/open-mmlab/mmsegmentation/pull/3300

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by CastleDream** [comment]
Claim the mission(has apply for this task in [超级视客营平台](https://openmmlab.com/activity/codecamp))
